Monika After Story/game/script-topics.rpy

```diff
@@ -5245,6 +5245,10 @@ label monika_penname:
     m 1eub "His real name is Charles Dodgson and he was a mathematician, but he loved literacy and word play in particular."
     m "He received a lot of unwanted attention and love from his fans, and he even received outrageous rumors."
     m 1ekc "He was somewhat of a one-hit wonder with his {i}Alice{/i} books but went downhill from there."
+    if seen_event("monika_1984"):
+        m 1esd "Also, if you remember me talking about George Orwell, that was also a pen name which he adopted in 1933 for his first publication named {i}Down and Out in Paris and London{/i}."
+        m 3eub "His actual name is Eric Blair, and before settling on his more famous pen name, he considered P.S. Burton, Kenneth Miles and H. Lewis Allways as alternatives."
+        m 1lud "One of the reasons he chose to publish his works under a pseudonym was to avoid embarrassment to his family over his time as a tramp."
```

Comment:
```suggestion
        m 1lksdlc "One of the reasons he chose to publish his works under a pseudonym was to avoid embarrassment to his family over his time as a tramp."
```